### PR TITLE
Calculator components

### PR DESCRIPTION
**Pull request by SamTush**
## What does this PR do?

This PR is raised to implement the following features: 👍 :

- [x] Create calculator component.
- [x] The application runs in the browser without errors 
- [x] No linter errors
- [x] Use Gitflow

## Description of the task to be completed ↓↓ 

- Create calculator component.

  
## How should it be tested

Clone the repository
```
git clone https://github.com/SamTush/math-magicians.git
```
Navigate to the folder
```
cd math-magicians
```
Checkout the branch
```
git checkout feature/create-components
```

## How should it be run

Run application in browser
```
run start
```

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+              </td>
+              <td>
+                <input className="btn all-btn orange-btn" type="button" value="=" />
+              </td>
+            </tr>
+          </table>
+        </div>
+      </div>
+    </div>
+  );
```
Comment (julie-ify) [inline src/components/calculator.js]: - [ ] The Calculator component should have at least one child component. Kindly add that.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @SamTush,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
- [ ] Kindly remove these files generated by the command as they would not be needed going forward.

jmhy

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SamTush** [comment]
Thank you for reviewing my PR @julie-ify 👍🏽. Cheers and Happy coding😎

**Review by Davidosky007 — APPROVED** [review]
Hi @SamTush 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Good looking readme file 👍 
* Proper use of Gitflow 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SamTush** [comment]
Thank you @Davidosky007 for reviewing my PR👍🏽. Have a wonderful day😎

**Pull request merged**